### PR TITLE
E2e readme docs

### LINKED ISSUE #14432: Update README file for e2e tests
**What**:
Update the README file for e2e testcases:
- explain how to run a single test
- fix the line about the chromedriver version is obsolete

**How**:
Following this example
https://github.com/trufflesuite/ganache#startup-options

### PR DESCRIPTION
**Pull request by seaona**
## Explanation
**What**:
Update the README file for e2e testcases:
- explain how to run a single test
- document options for running tests
- fix the line about the chromedriver version is obsolete

**How**:
Include table for flags, like [this](https://github.com/trufflesuite/ganache#startup-options) example

**Final View:**
image

Closes #14432

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -60,9 +60,24 @@ You can run the linter by itself with `yarn lint`, and you can automatically fix
 
 Our e2e test suite can be run on either Firefox or Chrome. In either case, start by creating a test build by running `yarn build:test`.
 
-Firefox e2e tests can be run with `yarn test:e2e:firefox`.
+- Firefox e2e tests can be run with `yarn test:e2e:firefox`.
 
-Chrome e2e tests can be run with `yarn test:e2e:chrome`, but they will only work if you have Chrome v79 installed. Update the `chromedriver` package to a version matching your local Chrome installation to run e2e tests on newer Chrome versions.
+- Chrome e2e tests can be run with `yarn test:e2e:chrome`. For avoiding potential errors, it is recommended to have the last version installed. 
+Update the `chromedriver` package to a version matching your local Chrome installation to run e2e tests on newer Chrome versions.
```
Comment (Gudahtt) [inline README.md:66]: This gives the impression that errors _might not_ happen if the versions don't match, but it reliably fails in that circumstance. It also assumes that `chromedriver` is _behind_ your local Chrome installation, but we've been pretty quick to update it lately so it ends up being _ahead_ of my local installation more often personally.

```suggestion
- Chrome e2e tests can be run with `yarn test:e2e:chrome`. The `chromedriver` package major version must match the major version of your local Chrome installation. If they don't match, update whichever is behind before running Chrome e2e tests.
```

Comment (seaona) [reply README.md:66]: that's better, thank you!!

**Review by Gudahtt — APPROVED** [review]
LGTM, thanks!

**Comment by Gudahtt** [comment]
Could you update this branch? The CI failure should be fixed on `develop`

**Review by jpuri — APPROVED** [review]

**Review by jpuri — APPROVED** [review]

**Pull request merged**